src/components/select/select.js

```diff
@@ -296,6 +326,15 @@ function SelectMenuDirective($parse, $mdUtil, $mdTheming) {
       }
     };
 
+    self.selectedLabels = function() {
+      var selectedOptionEls = Array.prototype.slice.call($element[0].querySelectorAll('md-option[selected]'));
+      if (selectedOptionEls.length) {
+        return selectedOptionEls.map(function(el) { return el.innerText; }).join(', ');
+      } else {
+        return undefined;
```

Comment:
Instead, this should return empty string. So it's always a string at least (Empty string is falsy remember)